qa/workunits/cephtool/test.sh

```diff
@@ -1355,6 +1355,7 @@ function test_mon_pg()
 
   ceph osd primary-affinity osd.0 .9
   expect_false ceph osd primary-affinity osd.0 -2
+  expect_false ceph osd primary-affinity osd.9999 .5
```

Comment:
the sha1 of the referenced commit will change after the merge, maybe you can squash this change into the one it is testing.